samples/snippets/set_agent_test.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+# We cannot test set_agent because Dialogflow ES can only have one agent
+# and if we create a agent it will delete the exisitng testing agent and
+# would cause all tests to fail
+def test_set_agent():
+    assert True
```

Comment:
Can you come up with a way for somehow making sure the request reaches to the server at least?
Here are some options:

1. use a different cloud project and actually create a new agent
2. use invalid argument and look at the error message
3. something else